source/common/http/status.cc

```diff
@@ -13,8 +13,10 @@ constexpr absl::string_view EnvoyPayloadUrl = "Envoy";
 
 absl::string_view statusCodeToString(StatusCode code) {
   switch (code) {
-  case StatusCode::Ok:
-    return absl::OkStatus().ToString();
+  case StatusCode::Ok: {
```

Comment:
Good find. Thanks. I think we can just simply return `"OK"` here.

Reply:
"OK" :)